pkgs/games/xpilot/default.nix

```diff
@@ -14,11 +14,11 @@
   zlib,
   libXxf86misc,
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "xpilot-ng";
   version = "4.7.3";
   src = fetchurl {
-    url = "mirror://sourceforge/xpilot/xpilot_ng/${pname}-${version}/${pname}-${version}.tar.gz";
+    url = "mirror://sourceforge/xpilot/xpilot_ng/${finalAttrs.pname}-${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

Comment:
I thought we didn't want to do finalAttrs.pname in FODs?

Reply:
Indeed, we usually try to not reuse `pname` and use a string literal instead.

I guess the author want to scope its PR and address those issue later on? Or should it be done within this PR? WDYT ?

Reply:
I think this PR should be closed -- this doesn't provide any improvement to `rec` at present, and increases e.g. thunk count.